english/marshmallow.md

```diff
@@ -280,7 +280,7 @@ Do you:
 
 [Make a Marshmallow cake](https://www.youtube.com/watch?v=3l1SY3QREv4)
 
-[Realize that you are Batman and have the Utility Belt on you. Yeah!!!](https://youtu.be/YpIQQeL2ZYk?t=22s)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.